src/translation-service.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module utils/translation-service
+ */
+
+const translations = {};
+
+/**
+ * Merges package translations to existing ones.
+ * These translations can be used later with {@link module:utils/translations-service~translate translate}
+ *
+ *		add( 'pl', {
+ *			'OK': 'OK',
+ *			'Cancel [context: reject]': 'Anuluj'
+ *		} );OK
+ *
+ * @param {String} lang
+ * @param {Object.<String, Object>} packageDictionary
```

Comment:
You don't know if this is a package's dictionary. These are translations and can come from any source.

This option also misses description and the type is wrong too (values are not objects).